engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/command/RenameSymbolCmd.scala

```diff
@@ -32,13 +32,13 @@ class RenameSymbolCmd(
       .map(module => Seq(new File(module.getPath)))
       .orElseGet(() => Seq())
 
-    val ensureCompiledJob = ctx.jobProcessor.run(
+    def ensureCompiledJob = ctx.jobProcessor.run(
```

Comment:
I assume this means that wherever we call `ensureCompiledJob` we schedule new `ctx.jobProcessor.run`. Yeah, that's certainly going to execute that job more frequently. What are the implications of doing that is something I can't easily estimate.